src/index.tsx

```
@@ -24,121 +1,144 @@
+	]);
 
-const client: ApolloClient<NormalizedCacheObject> = new ApolloClient({
-  cache: new InMemoryCache(),
-  link: combinedLink,
-});
-const fallbackLoader = <div className="loader"></div>;
+	const client: ApolloClient<NormalizedCacheObject> = new ApolloClient({
+		cache: new InMemoryCache(),
+		link: combinedLink,
+	});
+	const fallbackLoader = <div className="loader"></div>;
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)